code/game/objects/items/tools/welder.dm

```diff
@@ -213,6 +213,34 @@
 	desc = "An advanced welder designed to be used in robotic systems."
 	toolspeed = 0.5
 
+/obj/item/weldingtool/research
+	name = "research welding tool"
+	desc = "A scratched-up welder that's been modified many times. Is it still the same tool?"
+	icon_state = "welder_research"
+	item_state = "welder_research"
+	belt_icon = "welder_research"
+	maximum_fuel = 40
+	toolspeed = 0.75
```

Comment:
```suggestion
	toolspeed = 0.75
```

as above

Reply:
Same as the cargo screwdriver, though it has advantages other than the speed as well (the lower flash brightness)